src/pages/index.tsx

```diff
@@ -1,12 +1,124 @@
+import Link from 'next/link';
+import { ExternalLink } from '@/components/layout/ExternalLink';
 import { Head } from '@/components/layout/Head';
-import { Header } from '@/components/layout/Header';
 import { SITE_DESCRIPTION } from '@/lib/constants';
 
 const Home = () => {
+  const faqs = [
+    {
+      question: 'What is Cove?',
+      answer: (
+        <>
+          Cove is a novel approach to contract verification that aims to solve the{' '}
+          <ExternalLink text='problems' href='https://github.com/ScopeLift/cove-backend#why' /> of
+          existing verification tools.
+        </>
+      ),
+    },
+    {
+      question: 'How does it work?',
+      answer: (
+        <>
+          Instead of requiring you to submit a large JSON file or flattened solidity code, Cove asks
+          for git information and framework settings (e.g. use forge with the{' '}
+          <code className='text-sm'>default</code> profile) for the contracts you want to verify.
+          For now it also asks for transaction hashes, but we plan to remove this in the future.
+        </>
+      ),
+    },
+    {
+      question: 'What are the limitations?',
+      answer: (
+        <>
+          Cove is in alpha, so it currently has a few limitations that will be lifted in the future:
```

Comment:
Copy tweaks:

This version of Cove is just an alpha test. We don't currently store or expose the contract verification data. **By trying out Cove, you're helping us learn and improve**. The current version also has limited functionality:

* Only forge projects are supported
* Only Solidity contracts are supported
* Cove does not yet trace calls or search for deployment transactions....(continue from here as-is)